integration-tests/config/periodic/periodic.pict

```diff
@@ -0,0 +1,29 @@
+#
+# This is file used for github periodic job
+#
+
+OCP: 4.19, 4.18, 4.17
+AUTH: github, gitlab
+ACS: new, hosted
+Registry: quay.io, artifactory, nexus
+TPA: new, hosted
+SCM: github, gitlab, bitbucket
+Pipeline: tekton, gitlabci, actions, jenkins
+
+# Constraints (copied from default.pict)
+## Parameter value validation. This doesn't work with the current pict docker image, we can uncomment it when we figure out it in future
+[ACS] IN {"new", "hosted"};
```

Comment:
could you use the values `remote` and `local`? This is to align with  the values defined in repo [tssc-test ](https://github.com/redhat-appstudio/tssc-test)